docker_images/fairseq/tests/test_api.py

```diff
@@ -10,6 +10,7 @@
 # models are recommended for faster tests.
 TESTABLE_MODELS: Dict[str, str] = {
     "text-to-speech": "facebook/fastspeech2-en-ljspeech",
+    "audio-to-audio": "facebook/xm_transformer_s2ut_800m-es-en-st-asr-bt_h1_2022",
```

Comment:
Note that this model is >9GB which is almost at the limit of the max size for the inference API (10GB). I tried to run this test but it crashed in my computer due to this

Reply:
This test case for audio to audio has been removed and it could run successfully on my local machine after waiting for model downloading.